common/djangoapps/third_party_auth/tests/testutil.py

```diff
@@ -114,6 +117,18 @@ def verify_user_email(cls, email):
         user.save()
 
     @staticmethod
+    def configure_oauth_client():
+        """ Configure a oauth client for testing """
+        return OAuth2Client.objects.create(client_type=constants.CONFIDENTIAL)
+
+    @staticmethod
+    def configure_api_permission(client, provider_id):
+        """ Configure the client and provider_id pair. This will give the access to a client for that provider. """
+        obj = ApiPermissions(client=client, provider_id=provider_id)
+        obj.save()
+        return obj
```

Comment:
Nit: To create and save an object in one step use the [create method](https://docs.djangoproject.com/en/dev/ref/models/querysets/#django.db.models.query.QuerySet.create).  So the 3 lines above would become:

`return ApiPermissions.objects.create(client=client, provider_id=provider_id)`